tests/test_ui.py

```diff
@@ -179,6 +196,24 @@ def addSample(self):
         widget_2 = self.getDockedWidget(self.window.docks, InsertPrimitiveDialog.dock_flag)
         self.assertIsNot(widget, widget_2)
         old_vertex_count = len(self.model.sample.vertices)
+
+        self.window.sample_properties_dialog_action.trigger()
+        sample_properties_widget = self.getDockedWidget(self.window.docks, SampleProperties.dock_flag)
+        self.assertTrue(isinstance(sample_properties_widget, SampleProperties))
+        bytes_used = self.model.sample.vertices.nbytes
+        bytes_to_mb_factor = 1 / (1024**2)
+        memory = bytes_used * bytes_to_mb_factor
+        memory_formatted = f'{memory:.4f}'
```

Comment:
you don't need this variable. Use the f-string directly its already clear that you are formatting the memory variable

Reply:
memory_formatted variable removed from test_widget.py and test_ui.py